horovod/tensorflow/xla_mpi_ops.cc

```diff
@@ -476,11 +497,19 @@ class XLAPersistentBuffer : public common::PersistentBuffer {
 XLAPersistentBuffer::XLAPersistentBuffer(int device, int64_t size)
     : device_(device) {
   int restore_device;
+#if HAVE_CUDA
   CUDA_CALL(cudaGetDevice(&restore_device));
   CUDA_CALL(cudaSetDevice(device));
   // Simply call cudaMalloc for persistent buffer.
   CUDA_CALL(cudaMalloc((void**)&buffer_, size));
   CUDA_CALL(cudaSetDevice(restore_device));
+#elif HAVE_ROCM
+  HVD_GPU_CHECK(hipGetDevice(&restore_device));
+  HVD_GPU_CHECK(hipSetDevice(device));
+  // Simply call cudaMalloc for persistent buffer.
```

Comment:
Nit: Use `hipMalloc` in the comment here

Reply:
Fixed in the latest commit.